engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/resolve/TypeSignatures.scala

```diff
@@ -252,7 +252,7 @@ case object TypeSignatures extends IRPass {
     }
   }
 
-  val anyIr = Name.Qualified(
+  def anyIr = Name.Qualified(
```

Comment:
How is this better than `val anyIr = ...`? `TypeSignatures` is `case object`, so it is a singleton. Which means, that there should be exactly one `anyIr` value. Refactoring this to method means that there will be multiple lists of `Name.Literal`.

By the way, it could be declared `private`.